2023/gaussian-convolution/spanish/sentence_translations.json

```diff
@@ -794,7 +794,7 @@
  {
   "input": "What that means is that as you apply this process of repeated convolutions, a Gaussian doesn't change, it's a fixed point.",
   "model": "nmt",
-  "translatedText": "Lo que eso significa es que al aplicar este proceso de convoluciones repetidas, un gaussiano no cambia, es un punto fijo.",
+  "translatedText": "Lo que eso quiere decir es que al aplicar este proceso de convoluciones repetidas, una gaussiana no cambia, es un punto fijo.",
```

Comment:
Same question as above: what is wrong with "significa"? I do not find it overly formal and it makes the sentence shorter.

Reply:
Suggestion committed [here](https://github.com/pabloasenjo3/captions/commit/3c1e3ff35db8edfd76a1577ef3ae9f91a36bf384).